### DISCUSSION TITLE
vdirsyncer failing after python3.9 upgrade

### DISCUSSION THREAD

**Discussion by jbbudzon** [Homebrew/discussions#91] [Everyday usage] [answered]
It seems like vdirsyncer was updated to use python3.9 in #62452 for #62201.

However, when running vdirsyncer after the update, I'm getting a python error `'xml.etree.ElementTree.Element' object has no attribute 'getiterator'`. It seems this method was deprecated in python3.9 but was available until python3.8.

If anyone can reproduce this issue, we can open a ticket to move vdirsyncer back to python3.8 until the project has been updated to support python3.9.

Thanks!!

**Comment by Rylan12** [comment]
A better solution is to open a PR upstream that enables Python 3.9 support. Once it's made, we can add a patch to apply the changes without necessarily needing a new release.

**Comment by dtrodrigues** [comment, marked as answer]
I'm unable to reproduce the issue locally, but Homebrew/homebrew-core#63467 pulls in the upstream change to support Python 3.9

**Comment by jbbudzon** [comment]
@dtrodrigues Thank you very much!! Perhaps I updated at an awkward time 😝  I pulled down the latest vdirsyncer and everything is working!!
